components/Button/index.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+
+import Button from '.';
+
+describe('Button', () => {
+  it('renders a button', () => {
+    const props = {
+      type: 'reset',
+      size: 'small',
+      dark: true,
+      dataTestId: 'reset_button',
+      children: 'reset',
+    };
+
+    render(<Button {...props} />);
```

Comment:
How about rendering the button with a label so that we can check that the label is rendered too? 

```suggestion
    const label = 'Button Label';

    render(<Button {...props} >{label}</>);
    
    //...
    
    expect(button).toHaveTextContent(label);
```

Reply:
added on d129258d877c1e23ba31313cc0390ad9627eb192